test/integration/rules/no-autoplay-audio/no-autoplay-audio.html

```diff
@@ -0,0 +1,42 @@
+<!-- Passed  -->
```

Comment:
Add test for audio/video element with `autoplay` attribute but no value: `<audio autoplay>`

Reply:
Add test case for audio/video element with `autoplay` attribute, a duration / playableDuration <3, and loop attribute (should fail)